helm/values.yaml

```diff
@@ -125,6 +125,10 @@ corda:
       identityManagerAddress: ""
       # networkmapAddress defines the accesspoint for the Network Map server (public domain name+port)
       networkmapAddress: ""
+      # compatibilityZoneEnabled set to true for testnet
+      compatibilityZoneEnabled: false
+      # compatibilityZoneURL The root address of Corda compatibility zone network management services
```

Comment:
compatibilityZoneURL defines the root address of Corda compatibility zone network management services. For Corda Testnet you can use "https://netmap.testnet.r3.com"